WTWSMS/common/nicknames/00_nicknames.txt

```diff
@@ -1925,8 +1949,10 @@ nick_the_wolf = {
 #################################
 nick_the_hammer = {
 	allow = {
-		trait = brave
-		trait = champion
+		OR = {
+			trait = brave
+			trait = champion
+		}
```

Comment:
It is yet again an unique nickname for those with those specific traits. If all nicknames are too common there won't be much in way of variety.